src/legacy/core_plugins/visualizations/public/np_ready/public/services.ts

```diff
@@ -17,12 +17,36 @@
  * under the License.
  */
 
-import { I18nStart, IUiSettingsClient } from 'src/core/public';
+import {
+  Capabilities,
+  HttpStart,
+  I18nStart,
+  IUiSettingsClient,
+  SavedObjectsStart,
+} from 'src/core/public';
 import { TypesStart } from './types';
 import { createGetterSetter } from '../../../../../../plugins/kibana_utils/public';
+import { IndexPatternsContract } from '../../../../../../plugins/data/public';
+import { UsageCollectionSetup } from '../../../../../../plugins/usage_collection/public';
 
 export const [getUISettings, setUISettings] = createGetterSetter<IUiSettingsClient>('UISettings');
 
+export const [getCapabilities, setCapabilities] = createGetterSetter<Capabilities>('Capabilities');
+
+export const [getHttp, setHttp] = createGetterSetter<HttpStart>('Http');
```

Comment:
We are defining these GettersSetters in multiple plugins.
Do you think it might make sense to move these definitions to kibana_utils?

Reply:
i don't have a strong feeling, but keeping them inside each plugin makes it clear what services a plugin is using.